tools/querytee/goldfish/end_to_end_test.go

```diff
@@ -0,0 +1,271 @@
+package goldfish
```

Comment:
love the end to end test, can we add a happy and sad path e2e for metric queries as well?